alpha/lib/model/UserLoginDataPeer.php

```diff
@@ -836,8 +853,9 @@ public static function addLoginData($loginEmail, $password, $partnerId, $firstNa
 		{
 			$userQuota = $partner->getAdminLoginUsersQuota();
 			$adminLoginUsersNum = $partner->getAdminLoginUsersNumber();
+			$excludedAdminRoleUsersNum = self::getExcludedAdminRoleUsersNumber($partner);
 			// check if login users quota exceeded - value -1 means unlimited
-			if ($adminLoginUsersNum  && (is_null($userQuota) || ($userQuota != -1 && $userQuota <= $adminLoginUsersNum))) {
+			if ($adminLoginUsersNum  && (is_null($userQuota) || ($userQuota != -1 && $userQuota <= $adminLoginUsersNum - $excludedAdminRoleUsersNum))) {
```

Comment:
better to create a new variable for $adminLoginUsersNum - $excludedAdminRoleUsersNum. the line will be shorter and more readable